internal/services/network/route_table_data_source.go

```diff
@@ -25,6 +25,11 @@ func dataSourceRouteTable() *pluginsdk.Resource {
 		},
 
 		Schema: map[string]*pluginsdk.Schema{
+			"disable_bgp_route_propagation": {
```

Comment:
We have a convention for the names of boolean properties so can we rename this to 
```suggestion
			"bgp_route_propagation_enabled": {
```
and also move this property down in the schema, since it's computed it should be defined after the required arguments.

Reply:
I wanted to reflect this property to be the same as it is in resource: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/internal/services/network/route_table_resource.go

Here we have "disable_bgp_route_propagation". I don't think we should change it to "bgp_route_propagation_enabled" - let's keep things consistent.

Reply:
Apologies for taking a while to circle back to this. 

It looks like the property in the resource was added quite a while ago, before the naming standard for booleans across the provider was introduced and began being enforced. These should be phased out and renamed - could you please add a comment above the property in the resource to the effect of 
`//TODO rename to bgp_route_propagation_enabled in 4.0`
And rename this property to `bgp_route_propagation_enabled`.

Reply:
I've applied fixes based on your feedback, thanks.